src/transformers/generation/configuration_utils.py

```diff
@@ -423,6 +427,7 @@ def __init__(self, **kwargs):
         # Assistant generation
         self.num_assistant_tokens = kwargs.pop("num_assistant_tokens", 5)
         self.num_assistant_tokens_schedule = kwargs.pop("num_assistant_tokens_schedule", "heuristic")
+        self.assistant_confidence_threshold = kwargs.pop("assistant_confidence_threshold", 0)
```

Comment:
```suggestion
        self.assistant_confidence_threshold = kwargs.pop("assistant_confidence_threshold", None)
```
Let's default to `None` -- it's easier for us to check whether a parameter is set 🤗 

(needs change in the docstring as well)